tests/snapshots/TestCLICommands_atmos_vendor_pull_(no_tty).stderr.golden

```diff
@@ -1,7 +0,0 @@
-INFO Vendoring from 'vendor.yaml'
-WARN No TTY detected. Falling back to basic output. This can happen when no terminal is attached or when commands are pipelined.
-INFO ✓ github/stargazers (main)
-INFO ✓ weather (main)
-INFO ✓ ipinfo (main)
-INFO Vendored 3 components.
-
```

Comment:
DEBUG < INFO , and if we're at DEBUG we should still see these messages. Why did these `INFO` messages go away?

Reply:
https://github.com/cloudposse/atmos/pull/1212#discussion_r2054418887

Reply:
This whole snapshot is deleted